### DISCUSSION TITLE
How does authentication exactly work in Connexion?

### DISCUSSION THREAD

**Discussion by FeryET** [spec-first/connexion#1688] [Q&A] [answered]
Hello.

I'm trying to wrap my head around how exactly one should implement authentication/authorization in connexion. Normally in a flask app I would use `flask-secuirty-too` or `flask-login` for session based authentication. In connexion apparently I should define verification functions for tokens/cookies. Is there anyway to use boilerplate flask extensions for these parts? The documentaion regarding `connexion.security` module is pretty ambigious. Plus, implementing basic boilerplate authenticaiton/authorization codes that are pretty repititive in all connexion projects seem to be a pretty weird choice.

Can I use flask extensions underneath connexion, and also have their specification in the `openapi.yaml` file?

**Comment by RobbeSneyders** [comment, marked as answer]
Please check our [updated security docs](https://connexion.readthedocs.io/en/latest/security.html).
